src/components/Permalink/Permalink.js

```diff
@@ -147,7 +152,7 @@ class Permalink extends Component {
     this.setState({
       x: this.roundCoord(center[0]),
       y: this.roundCoord(center[1]),
-      z: mapView.getZoom(),
+      z: Permalink.roundToTwo(mapView.getZoom()),
```

Comment:
The function just reduces readability. Do the rounding here, please.

Reply:
I removed the function as you prefer